websockify/token_plugins.py

```diff
@@ -87,3 +87,45 @@ class JSONTokenApi(BaseTokenAPI):
     def process_result(self, resp):
         resp_json = resp.json()
         return (resp_json['host'], resp_json['port'])
+
+
+class JWTTokenApi(BasePlugin):
+    # source is a JWT-token, with hostname and port included
+    # Both JWS as JWE tokens are accepted. With regards to JWE tokens, the key is re-used for both validation and decryption.
+
+    def lookup(self, token):
+        from jwcrypto import jwt
```

Comment:
I'd catch `ImportError` here as well and handle it with a proper error message

Reply:
Oh, yes, should've done this right away, will fix! Thanks!

Reply:
Done and done!